app/service/interfaces/i_login_handler.py

```diff
@@ -0,0 +1,32 @@
+import abc
+
+from app.utility.base_object import BaseObject
+
+
+class LoginHandlerInterface(abc.ABC, BaseObject):
+    def __init__(self, name):
+        self._name = name
```

Comment:
shouldn't this be 'services' (using DefaultLoginHandler as the reference)

Reply:
You mean passing in services into the constructor?